README.rst

```diff
@@ -1079,6 +1079,53 @@ Assert that the request was called exactly n times.
         responses.assert_call_count("http://www.example.com?hello=world", 1) is True
 
 
+Assert Request Calls data
+------------------
+
+``Request`` object has ``calls`` list which elements correspond to ``Call`` objects
+in the global list of ``Registry``.
```

Comment:
Need more context on the issue that it solves for the user rather than the statement of technical fact

Reply:
I have extended description and replaced the example, please review.